ls1intum/Artemis
# Fixed two modeling assessment bugs

### Checklist


### Motivation and Context



### Description

1: Fixed bug where submission for a modeling assessment did not show up, the assessment was locked by null. (for not exam mode exercises)
2: Fixed bug where instructor could override wrong results. 

### Steps for Testing

1: Create a modelingexercise, participate and assess. If assessing works the bug is fixed.

2: Create an exam with two correction rounds, assess a submission in both rounds. Then try to override the result of the first round, it should be overridden correctly, and not the second result.
